devito/symbolics/extended_dtypes.py

```diff
@@ -15,6 +18,62 @@
 }
 
 
+class NoDeclStruct(ctypes.Structure):
```

Comment:
@ZoeLeibowitz I may be wrong but don't you need this type too? please see also where it's used and comment

Reply:
We have different use cases because I need a local composite type which still generates a struct in the header file (i.e still produces a struct definition)